cmd/errtrace/main_test.go

```diff
@@ -391,16 +391,18 @@ func extractLogs(src []byte) ([]logLine, error) {
 	return logs, nil
 }
 
-func parseLogOutput(s string) ([]logLine, error) {
+func parseLogOutput(file, s string) ([]logLine, error) {
 	var logs []logLine
 	for _, line := range strings.Split(s, "\n") {
 		if line == "" {
 			continue
 		}
 
+		// Drop the path so we can determinstically split on ":" (which is a valid character in Windows paths).
+		line = strings.TrimPrefix(line, file)
 		parts := strings.SplitN(line, ":", 4)
 		if len(parts) != 4 {
-			return nil, fmt.Errorf("bad log line: %q", line)
+			return nil, fmt.Errorf("bad log line1: %q", line)
```

Comment:
"line1"?

Reply:
removed, debugging leftover (to identify which "bad log line" print was causing the issue)